packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -685,24 +685,18 @@ function updateOffscreenComponent(
 ) {
   const nextProps: OffscreenProps = workInProgress.pendingProps;
   const nextChildren = nextProps.children;
+  const isPendingDetached = (workInProgress.stateNode._pendingVisibility & OffscreenDetached) !== 0;
```

Comment:
Nit: maybe `nextIsDetached` for consistency?

Reply:
Good idea!